manifest.json

```diff
@@ -18,7 +18,7 @@
         "default_title": "JIRA Linker"
     },
     "content_scripts": [{
-        "matches": ["https://github.com/*"],
+        "matches": ["https://github.com/*/*/pull/*/files"],
```

Comment:
This will make it not work for individual commits (e.g. `https://github.com/janepeng/github_helper/pull/4/commits/574da70c70df9c304b14f83b4e45ce014221bd54`)